runners/core-construction-java/src/test/java/org/apache/beam/runners/core/construction/SdkComponentsTest.java

```diff
@@ -124,6 +124,18 @@ public void leaveCompositeTransform(Node node) {
                   equalTo(windowingStrategies.size()));
             } else {
               transforms.add(node);
+              if (PTransformTranslation.COMBINE_TRANSFORM_URN.equals(
+                  PTransformTranslation.urnForTransformOrNull(node.getTransform()))) {
+                // Combine translation introduces a coder that is not assigned to any PCollection
+                // in the default expansion, and must be explicitly added here.
+                try {
+                  addCoders(
+                      CombineTranslation.getAccumulatorCoder(
```

Comment:
It does; it doesn't show in the default expansion of Combine (because we don't group accumulators) but we do translate it (because it gets tossed in the payload)

I'll update the comment.